paper.bib

```diff
@@ -0,0 +1,77 @@
+@misc{frontier,
+    author = {Barb Helland},
+    title = {At the Frontier: DOE Supercomputing Launches the Exascale Era},
+    year = {2022},
+    publisher = {United States Department of Energy Office of Science},
+    journal = {Website},
+    url = {https://www.energy.gov/science/articles/frontier-doe-supercomputing-launches-exascale-era}
+}
+
+@misc{official_solution,
+    author = {tjwrona1992},
+    title = {What is the proper way to document a cmake module},
+    year = {2019},
+    publisher = {StackOverflow},
+    journal = {Stack Overflow Forum},
+    url = {https://stackoverflow.com/a/54671996}
+}
+
+@misc{sphinx_plugin,
+    author = {Kitware},
+    title = {Sphinx Domain for Modern CMake},
+    year = {2022},
+    publisher = {GitHub},
+    journal = {GitHub Repository},
```

Comment:
I think this renders badly in the resulting paper, although the JOSS example paper does it like that. I would propose to use
```suggestion
    howpublished = {{GitHub Repository}},
```